### ISSUE TITLE
Finish the implementation of set operators in the simulator

### ISSUE THREAD

**Issue by konnov** [informalsystems/quint#238]
A few challenging set operators are missing in the simulator. This issue keeps progress on them:

 - [x] `S.powerset()`, see #148
 - [ ] `S.allLists()`, depends on #231
 - [ ] `S.chooseSome()`, see #150

**Comment by konnov** [comment]
This is actually a duplicate of #136
